.oscal/ssp.oscal.yaml

```diff
@@ -0,0 +1,289 @@
+          - name: job-title
+            value: Software Architect
+      - uuid: 22222222-0000-4000-9000-100000000002
+        type: person
+        name: "Paul Wand"
+        props:
+          - name: job-title
+            value: Senior Software Architect
+      - uuid: 88888888-0000-4000-9000-100000000008
+        type: person
+        name: "John Doe"
+        props:
+          - name: job-title
+            value: Document Preparer
+      - uuid: 99999999-0000-4000-9000-100000000009
+        type: organization
+        name: "FedRAMP PMO"
+        addresses:
+          - addr-lines:
+              - "1234 FedRAMP Way"
+            city: "Washington"
+            state: "DC"
+            postal-code: "20001"
+
+    responsible-parties:
+      - role-id: system-owner
+        party-uuids:
+          - 11111111-0000-4000-9000-100000000002
+      - role-id: information-security-officer
+        party-uuids:
+          - 22222222-0000-4000-9000-100000000002
+      - role-id: prepared-by
+        party-uuids:
+          - 88888888-0000-4000-9000-100000000008
+      - role-id: prepared-for
+        party-uuids:
+          - 99999999-0000-4000-9000-100000000009
+
+    props:
+      - name: marking
+        value: "SENSITIVE BUT UNCLASSIFIED"
+
+  import-profile:
+    href: "profile.oscal.yaml"
+
+  system-characteristics:
+    system-ids:
+      - id: "ESC-DEV-001"
+        identifier-type: https://fedramp.gov
+    system-name: "Enterprise Software Development Environment"
+    system-name-short: "ESDE"
+    description: >-
+      Primary software development environment for Example Software Company,
+      including development, testing, and build infrastructure.
+    security-sensitivity-level: fips-199-moderate
+    system-information:
+      information-types:
+        - title: "Software Development Information"
+          uuid: 370026f8-582b-4c02-9331-a5c84afdbfd8
+          description: "Source code, build configurations, and development artifacts"
+          confidentiality-impact:
+            base: fips-199-moderate
+            selected: fips-199-moderate
+            adjustment-justification: "Default FIPS-199 moderate impact"
+          integrity-impact:
+            base: fips-199-moderate
+            selected: fips-199-moderate
+            adjustment-justification: "Default FIPS-199 moderate impact"
+          availability-impact:
+            base: fips-199-moderate
+            selected: fips-199-moderate
+            adjustment-justification: "Default FIPS-199 moderate impact"
+    
+    security-impact-level:
+      security-objective-confidentiality: moderate
+      security-objective-integrity: moderate
+      security-objective-availability: moderate
+
+    status:
+      state: operational
+
+    authorization-boundary:
+      description: "The authorization boundary encompasses all core development systems"
+    
+    network-architecture:
+      description: "Network architecture showing system components and connections"
```

Comment:
This is ok for now, but we will need to think about how we handle this in the long-term for an application and supporting libraries that do not have an authorization boundary and network architecture _out of the box_.

Reply:
added better ones